app/views/courses/financial_support/_advice.html.erb

```diff
@@ -0,0 +1,2 @@
+<p class="govuk-body">You may be eligible for a <a href="https://getintoteaching.education.gov.uk/funding-my-teacher-training/tuition-fee-and-maintenance-loans">loan while you study</a> – note that you’ll have to apply for <a href="https://www.gov.uk/student-finance">undergraduate student finance</a>.</p>
+<p class="govuk-body">Find out about financial support if you’re from <a href="https://getintoteaching.education.gov.uk/explore-my-options/overseas-graduates">outside the UK</a>.</p>
```

Comment:
Lets use the `govuk_link` helper for these links – means they get the right styling.

Reply:
Actually, scratch that. We shouldn’t need any new partials as we should be using those we already have in `app/views/courses/financial_support` folder, partials which are shown depending on the course. Where ever the logic for that is, I suspect that’s where the new `AdviceComponent` should be used.